src/plugins/vis_augmenter/public/view_events_flyout/components/utils.tsx

```diff
@@ -51,44 +90,15 @@ function getValueAxisPositions(embeddable: VisualizeEmbeddable): { left: boolean
  * @param setVisEmbeddable custom hook used in base component
  * @param setErrorMessage custom hook used in base component
  */
-export async function fetchVisEmbeddable(
+export async function fetchVisEmbeddableWithSetters(
   savedObjectId: string,
   setTimeRange: Function,
   setVisEmbeddable: Function,
   setErrorMessage: Function
 ): Promise<void> {
-  const embeddableVisFactory = getEmbeddable().getEmbeddableFactory('visualization');
   try {
-    const contextInput = {
-      filters: getQueryService().filterManager.getFilters(),
-      query: getQueryService().queryString.getQuery(),
-      timeRange: getQueryService().timefilter.timefilter.getTime(),
-    };
-    setTimeRange(contextInput.timeRange);
-
-    const embeddable = (await embeddableVisFactory?.createFromSavedObject(savedObjectId, {
-      ...contextInput,
-      visAugmenterConfig: {
-        inFlyout: true,
-        flyoutContext: VisFlyoutContext.BASE_VIS,
-      },
-    } as VisualizeInput)) as VisualizeEmbeddable | ErrorEmbeddable;
-
-    if (embeddable instanceof ErrorEmbeddable) {
-      throw getErrorMessage(embeddable);
-    }
-
-    embeddable.updateInput({
-      // @ts-ignore
-      refreshConfig: {
-        value: 0,
-        pause: true,
-      },
-    });
-
-    // By waiting for this to complete, embeddable.visLayers will be populated
-    await embeddable.populateVisLayers();
-
+    const embeddable = await fetchVisEmbeddable(savedObjectId);
+    setTimeRange(embeddable.timeRange);
```

Comment:
For consistency, could we force this to be the same as it was before?
```
setTimeRange(getQueryService().timefilter.timefilter.getTime());
```
this means we don't have to make the private field public in embeddable as well. Plus if we move away from embeddables altogether, we will inevitably have to make this change.